packages/cli/src/commands/generate/helpers.ts

```diff
@@ -62,10 +74,13 @@ export const pathName = (path, name) => {
 export const createYargsForComponentGeneration = ({
   componentName,
   filesFn,
-}) => {
+}: {
+  componentName: 'cell' | 'component' | 'function' | 'layout' | 'service'
+  filesFn: Function
+}): CommandModule => {
   return {
     command: `${componentName} <name>`,
-    desc: `Generate a ${componentName} component.`,
+    describe: `Generate a ${componentName} component.`,
```

Comment:
yargs type says this is the correct way to call this member. Does this break something?

Reply:
They have multiple aliases: 
> desc/describe/description: string, the option description for help content, see describe()

https://yargs.js.org/docs/#api-optionskey-opt, I'm inclined to go with the type definition here